content/blog/2021/12/camel314-whatsnew/index.md

```diff
@@ -0,0 +1,141 @@
+Now you can configure a number of route configurations, and then specify on each route which configuration to use (you can use match by ids, wildcards, and regular expression).
+
+### Reload Routes
+
+The route reload functionality in Camel is capable of watching a directory folder for file changes, and then automatically trigger a reload of the running routes in the Camel application.
+This functionality is intended for development purposes, and not for production use.
+
+You can see the reloading in action from one of the examples such as: https://github.com/apache/camel-examples/tree/master/examples/main-xml
+
+### Kamelets
+
+Kamelets is a higher level building blocks that we keep innovating and improve over the coming releases.
+For Camel 3.14 we worked on making Kamelets more configurable, and more self-contained.
+
+A Kamelet is now capable of configuring its own component instances which avoids clashing
+with other kamelets or regular components (local beans). This ensures a Kamelet is a _black box_.
+
+We made using `AggregationStrategy` easier with scripting languages, which allows
+for example Kamelets to embed code as the strategy directly in the Kamelet spec file.
+
+### Health Checks
+
+Camel components can now provide custom health checks which can be automatically discovered by Camel.
+
+We added new health checks for route consumers so a consumer can more preciesly
+report health status, and in case of errors, then the consumer reports more detailed
+information about the cause such as exception, and http status codes etc.
+
+Camel now also ensures scheduled based consumers, is run at least once during startup
+before the health check can be regarded as UP. Previously what could happen is that Camel
+would report UP during startup, and then on first run by the consumer, it is causing an
+error and then the health check status would soon after be flipped to DOWN.
+
+There are now also more configurations to fine-tune the health checks such as number of
+successive calls before a reporting as either UP or DOWN (flaky checks).
+
+We will continue to improve health checks in the upcoming releases.
+
+### JBang
+
+CamelJBang is used to quickly try and run Camel applications.
+
+[JBang](https://www.jbang.dev/) makes it very easy to run Java without all the ceremony.
+With CamelJBang we made it possible for anyone to quickly try Camel.
+
+The Camel Karavan Designer uses CamelJBang in its editor that allows you to visually
+build Camel integrations and run them with ease while designing directly from the editor or web-browser.
+
+### Kafka
+
+The `camel-kafka` component has been overhauled and optimized to be more robust and have better performance.
+
+We also worked on making _resume from offset_ a general concept in Camel and provide
+more out-of-the-box strategies for `camel-kafka`.
```

Comment:
Here's my suggestion: 

The `camel-kafka` component has been overhauled and optimized to be more robust, have better
 [performance]((https://camel.apache.org/blog/2021/09/camel-kafka-consumer-changes) and 
 improve resource utilization. Users of the Kafka idempotent repository will also benefit from 
 some of the robustness fixes implemented during the last releases. 

Additionally, added support for async commits, we increased the test coverage, and included 
forward testing support for Kafka 3.0. 

We also worked on making _resume from offset_ a general concept in Camel and provide more 
out-of-the-box strategies for `camel-kafka`.

Reply:
Thanks. To use links on the website is a pain as you cannot use absolute links and the xref stuff is "hit and miss" to figure out, so its often easier to not have any links.